src/EditorFeatures/CSharpTest/QuickInfo/SyntacticQuickInfoSourceTests.cs

```diff
@@ -293,15 +293,8 @@ private IQuickInfoProvider CreateProvider(TestWorkspace workspace)
             Assert.NotNull(state);
 
             var viewHostingControl = (ViewHostingControl)((ElisionBufferDeferredContent)state.Content).Create();
-            try
-            {
-                var actualContent = viewHostingControl.ToString();
-                Assert.Equal(expectedContent, actualContent);
-            }
-            finally
-            {
-                viewHostingControl.TextView_TestOnly.Close();
```

Comment:
❓ Can you not close the control, and then verify that the buffer is disconnected?